### DISCUSSION TITLE
tor function possible

### DISCUSSION THREAD

**Discussion by sketax** [isontheline/pro.webssh.net#353] [Ideas]
Hey,

it is possible to implement a tor functionality?
Would be great to connect with SSH over an onion adress.

ty :)

**Comment by isontheline** [comment]
Hey @sketax 👋

It's possible but :
* Does a TOR client exists on iOS with a MIT license?
* Why not use a VPN instead?
* What about countries regulation? WebSSH can't be purchased in some countries because implementing a VPN, I won't reduce WebSSH availability in other countries again.

Could you tell me your use case?

Have a nice day ☀️

**Reply by sketax** [reply]
I want to connect to my node with ssh. As long as i don’t want my clearnet ip published I have to use tor.
There isn’t any application on the whole iOS ecosystem which supports both. I know a plenty of people in the btc-scene who want exactly this feature.

**Reply by isontheline** [reply]
OK that make sense.

Could you open an issue to track progress?
